### PR TITLE
HS-951 Merge Flows-Adapter module into Flows-Parser

### PR DESCRIPTION
**Pull request by FrancescoBuc86**
https://opennms.atlassian.net/browse/HS-951

### CODE REVIEW

File: minion/minion-flows/flows-parser/src/main/java/org/opennms/horizon/minion/flows/adapter/imported/PipelineImpl.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.horizon.minion.flows.adapter.imported;
+
+import java.util.List;
+
+import org.opennms.horizon.minion.flows.adapter.common.FlowException;
+
+public class PipelineImpl implements Pipeline {
```
Comment (freddy61025) [inline minion/minion-flows/flows-parser/src/main/java/org/opennms/horizon/minion/flows/adapter/imported/PipelineImpl.java:35]: Do you really need this ?

Comment (freddy61025) [reply minion/minion-flows/flows-parser/src/main/java/org/opennms/horizon/minion/flows/adapter/imported/PipelineImpl.java:35]: Also Pipeline.java

File: minion/minion-flows/flows-parser/src/main/resources/OSGI-INF/blueprint/blueprint.xml [hunk 2]
```diff
@@ -86,4 +90,37 @@
         </service-properties>
     </service>
 
+    <!-- Netflow5 Factory and Adapter -->
+    <bean id="netflow5Factory" class="org.opennms.horizon.minion.flows.adapter.netflow5.Netflow5AdapterFactory">
```
Comment (freddy61025) [inline minion/minion-flows/flows-parser/src/main/resources/OSGI-INF/blueprint/blueprint.xml:94]: better to rename like netflow5AdapterFactory

**Review by freddy61025 — CHANGES_REQUESTED** [review]
please try remove extra files. e.g. EnrichedFlowProtos.java is that really needed for adapter?

File: minion/minion-flows/flows-parser/src/main/java/org/opennms/horizon/minion/flows/adapter/imported/PipelineImpl.java (hunk 1 above)
Comment (FrancescoBuc86) [reply minion/minion-flows/flows-parser/src/main/java/org/opennms/horizon/minion/flows/adapter/imported/PipelineImpl.java:35]: No, but I'm not modifying  anything here, just moving the classes, in order not to create confusion and lose track of the changes. These are done in here: https://github.com/OpenNMS-Cloud/horizon-stream/pull/690

File: minion/minion-flows/flows-parser/src/main/resources/OSGI-INF/blueprint/blueprint.xml (hunk 2 above)
Comment (FrancescoBuc86) [reply minion/minion-flows/flows-parser/src/main/resources/OSGI-INF/blueprint/blueprint.xml:94]: ok, but in the next PR

File: minion/minion-flows/flows-parser/src/main/java/org/opennms/horizon/minion/flows/adapter/imported/PipelineImpl.java (hunk 1 above)
Comment (FrancescoBuc86) [reply minion/minion-flows/flows-parser/src/main/java/org/opennms/horizon/minion/flows/adapter/imported/PipelineImpl.java:35]: ok, I wanted to leave all the changes to the next PR, but ok, I can update this one instead

**Review by freddy61025 — APPROVED** [review]
ok lets clean up in next PR then.

**Comment by FrancescoBuc86** [comment]
> ok lets clean up in next PR then.

No worries, I worked your comments in this pr anyway, I'm getting some trouble in getting

**Pull request merged**